### ISSUE TITLE
File mkdocs.yml is also copied during the serve and build process

### ISSUE THREAD

**Issue by froschdesign** [jdoiro3/mkdocs-multirepo-plugin#9]
The file `mkdocs.yml` of the imported repository is also copied to the dev server and to the output directory of the build process.

Example:

* `site`
  * `Imported`
    * `index.html`
    * `mkdocs.yml`

The configuration file should not be included in any output.

**Comment by jdoiro3** [comment]
@froschdesign, do you think it makes sense to change the location of the config file to outside the `docs` directory and change the name to `multirepo.yml`? This config file will contain the `nav` section contained in the imported repo.

*imported repo*
```
.
├── docs
│   └── index.md
└── multirepo.yml
```

**Comment by jdoiro3** [comment]
This issue was resolved in release `v0.3.2`. I also added the ability to store the config file for the plugin in the project's root directory and using a different name other than *mkdocs.yml* (e.g., *multirepo.yml*).

*imported repo with config in root directory*
```text
.
├── docs
│   └── index.md
└── multirepo.yml (config with nav or other config info for the plugin)
└── mkdocs.yml
```

*imported repo with config in docs directory*
```text
.
├── docs
│   └── index.md
│   └── multirepo.yml
└── mkdocs.yml
```

Both ways should work with the below main repo configuration.

*parent repo's mkdocs.yml*
```yaml
site_name: Test

nav:
  - Home: index.md
  - Section: '!import {url}?config=multirepo.yml'

plugins:
  - multirepo
```
